### DISCUSSION TITLE
关于同步的建议

### DISCUSSION THREAD

**Discussion by EDDIEFUN** [remotely-save/remotely-save#568] [Ideas]
是否增加自动同步的范围，除了开启同步外，增加一个关闭软件自动同步的功能？

**Comment by fyears** [comment]
自动同步，现在是一个开关喔？可以关闭的

**Reply by EDDIEFUN** [reply]
可能没有表达清楚。我的意思是：增加一个【关闭软件时自动同步】

**Reply by fyears** [reply]
不行的。因为这插件相当于运行在浏览器里，关掉就没了，无法后台运行

**Comment by Tmerowr** [comment]
win端和安卓手机端实现了同步，mac端同步不过去？有什么解决办法？用的onedrive

**Reply by fyears** [reply]
有什么症状呢？什么叫同步不过去呢

**Comment by EDDIEFUN** [comment]
'你好。已收到你的邮件，我会在假期结束后尽快处理。————————————这是来自QQ邮箱的自动回复邮件。'
